locales/en.json

```diff
@@ -28,7 +28,8 @@
     "app_location_unavailable": "This url is not available or conflicts with the already installed app(s):\n{apps:s}",
     "app_manifest_invalid": "Invalid app manifest: {error}",
     "app_no_upgrade": "No apps to upgrade",
-    "app_not_upgraded": "The following apps were not upgraded: {apps}",
+    "app_not_upgraded": "The following apps were not upgraded because a the app '{app}' failed to upgrade: {apps}",
+    "app_upgrade_stoped": "The upgrade of alls applications has been stopped to prevent possible dommages because the previous application failed to upgrade",
```

Comment:
```suggestion
    "app_upgrade_stoped": "The upgrade of all applications has been stopped to prevent possible damage because the previous application failed to upgrade",
```